clippy_lints/src/mutable_debug_assertion.rs

```diff
@@ -77,14 +77,14 @@ fn extract_call<'a, 'tcx>(cx: &'a LateContext<'a, 'tcx>, e: &'tcx Expr) -> Optio
                         if let ExprKind::Tup(ref conditions) = headerexpr.kind;
                         if conditions.len() == 2;
                         then {
-                            if let ExprKind::AddrOf(_, ref lhs) = conditions[0].kind {
+                            if let ExprKind::AddrOf(_, _, ref lhs) = conditions[0].kind {
```

Comment:
Maybe the ones in this file?

Reply:
eh, i think this is edge casey enough to leave it focused on Ref.